src/seedu/addressbook/ui/TextUi.java

```diff
@@ -164,5 +165,17 @@ private static String getIndexedListForViewing(List<String> listItems) {
     private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
```

Comment:
Correct implementation of the varags method. As discussed in tutorial, there are many possible placement for this method.